kratos/includes/element.h

```diff
@@ -10,30 +10,21 @@
 //  Main authors:    Pooyan Dadvand
 //
 
-
 #if !defined(KRATOS_ELEMENT_H_INCLUDED )
 #define  KRATOS_ELEMENT_H_INCLUDED
 
 // System includes
-#include <string>
-#include <iostream>
-#include <sstream>
-#include <cstddef>
 
 // External includes
 
 // Project includes
-#include "includes/define.h"
-#include "includes/node.h"
-#include "geometries/geometry.h"
 #include "includes/properties.h"
 #include "includes/process_info.h"
 #include "includes/geometrical_object.h"
-#include "utilities/indexed_object.h"
 #include "containers/flags.h"
-#include "containers/weak_pointer_vector.h"
```

Comment:
I think that this is needed to correctly define the NEIGHBOUR_CONDITIONS

Reply:
But NEIGHBOR_CONDITIONS is a variable right? Shouldn’t this then include it?
Compiles fine in the CI

Reply:
@philbucher the problem is the definition of that variable. right now it is done in the header.

examples of this are NEIGHBOUR_NODES, NEIGHBOUR_ELEMENTS, NEIGHBOUR_CONDITIONS defined in node.h, element.h, condition.h (all of these are WeakPointerVectors)

Reply:
hm I still don't get it (also it is not included in `Condition` btw) but you seems sure so I brought it back

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/condition.h#L1257

am i getting you wrongly?

Reply:
Ah there it is, didn't see this, sorry
I will add the include also to `condition.h` then